docs/content/preview/migrate/migrate-steps.md

```diff
@@ -224,6 +224,30 @@ Refer to [export data](../reference/yb-voyager-cli/#export-data) for details abo
 
 The options passed to the command are similar to the [`yb-voyager export schema`](#export-schema) command. To export only a subset of the tables, pass a comma-separated list of table names in the `--table-list` argument.
 
+#### Accelerate data export for MySQL and Oracle
+
+For MySQL and Oracle, you can optionally speed up data export by setting the environment variable `EXPERIMENTAL_FAST_EXPORT=1` when you run `export data` using yb-voyager.
+
+Consider the following caveats before using the feature:
+
+- You need to perform additional steps when you [prepare the source database](#prepare-the-source-database).
+- Some data types are unsupported. For a detailed list, refer to [datatype mappings](../reference/datatype-mapping-mysql/).
+- [--parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument will have no effect.
+- SSL is unsupported.
+- In MySQL RDS, writes are not allowed during the data export process.
+- For Oracle where sequences are not attached to a column, resume value generation is unsupported with EXPERIMENTAL_FAST_EXPORT=1 environment variable.
```

Comment:
Can you pls modify it to `For Oracle where sequences are not attached to a column, resume value generation is unsupported` 

`with EXPERIMENTAL_FAST_EXPORT=1 environment variable` is redundant given that it is under that section.